editor/import_dock.h

```diff
@@ -66,6 +66,7 @@ class ImportDock : public VBoxContainer {
 	void _importer_selected(int i_idx);
 	void _update_options(const Ref<ConfigFile> &p_config = Ref<ConfigFile>());
 	void _update_preset_menu();
+	void _add_keep_import_option(const String &importer_name);
```

Comment:
Our convention is to use `p_` prefix for parameters (or `r_` when they're mutated by the function).
```suggestion
	void _add_keep_import_option(const String &p_importer_name);
```